disconnected/updating/index.adoc

```diff
@@ -1,42 +1,36 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="about-restricted-network-updates"]
+[id="about-disconnected-updates"]
 = About cluster updates in a disconnected environment
 include::_attributes/common-attributes.adoc[]
-:context: about-restricted-network-updates
+:context: about-disconnected-updates
 
 toc::[]
 
-////
-WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to this assembly for earlier versions should be done in separate PRs based off of their respective version branches. Otherwise, your cherry picks may fail.
-
-To do: Remove this comment once 4.13 docs are EOL.
-////
-
 A disconnected environment is one in which your cluster nodes cannot access the internet.
 For this reason, you must populate a registry with the installation images.
 If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap.
 If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
```

Comment:
This text maybe came in via ccdbf035f56e480c , so orthogonal to this pull if  you want to punt.  But I was giving it a closer look as the lead-off for the new sub-section, and it seems like we could simplify.  "registry host" seems to be talking about a local host that can run `oc` mirroring commands?  Maybe something like:

> A disconnected environment is one in which your cluster nodes cannot access the internet, or where you want to manage update recommendations and release images locally for policy or performance purposes.  This section covers mirroring OpenShift Container Platform images, managing an OpenShift Update Service, and performing cluster updates in a disconnected environment.

or some such, that leaves details about mirroring to the subsection that covers mirroring?

Reply:
I think that sort of rewording is a good idea, especially since, once the [installation section's mirroring content](https://docs.openshift.com/container-platform/4.16/installing/disconnected_install/index.html) is migrated to this section, I'd like to either delete the OTA mirroring page or leave it as a stub that points to the main mirroring content.